cmd/sys-mgmt-executor/main.go

```diff
@@ -120,26 +128,36 @@ func checkDockerContainerStatus(service string, running bool) (bool, error) {
 	}
 }
 
-func ExecuteDockerCommands(service string, operation string) error {
+func ExecuteDockerCommands(service string, operation string) (string, error) {
 	// don't run commands for unknown services - could be an attack
 	if agent.IsKnownServiceKey(service) {
-		return runDockerCommands(service, operation)
+
+		LoggingClient.Info(fmt.Sprintf("4. IsKnownServiceKey() check a success for service %s", service))
+		LoggingClient.Info(fmt.Sprintf("5. operation: %s", operation))
+
+		_, err := runDockerCommands(service, operation)
+		if err != nil {
+			LoggingClient.Error(fmt.Sprintf("ExecuteDockerCommands() failed with %s\n", err.Error()))
+		}
 	}
-	return fmt.Errorf("the service %s is an unknown service for which request was made to run Docker command", service)
+	return "", fmt.Errorf("the service %s is an unknown service for which request was made to run Docker command", service)
 }
 
-func runDockerCommands(service string, operation string) error {
+func runDockerCommands(service string, operation string) (string, error) {
+
+	var outStr = ""
+	var errStr = ""
```

Comment:
You don't use these variables until the end of the function. You could eliminate these declarations and just do the following below.
`outStr, errStr := string(stdout), string(stderr)`